api.planx.uk/webhooks/sanitiseApplicationData/operations.ts

```diff
@@ -52,6 +55,47 @@ export const operationHandler = async (
   return operationResult;
 };
 
+/**
+ * Return list of session IDs which are now ready for sanitation
+ */
+export const getExpiredSessionIds = async (): Promise<string[]> => {
```

Comment:
This will find all sessions submitted more than 6 months ago, and iterate over them to delete files.

`s3.deleteObjects()` will not fail if it's passed an array of already deleted files (e.g. when this runs the second time).

Once this is live and has succeeded once, we can push a small change here to also check the `sanitised_at` column to ensure this operation only runs on...
 - Sessions submitted more than 6 months ago
 - Session not already flagged as sanitised (handled by the `sanitiseLowcalSessions` operation)